common/src/main/scala/org/neo4j/spark/service/SchemaService.scala

```diff
@@ -436,7 +434,10 @@ class SchemaService(
       queryReadStrategy.createStatementForNodeCount(options)
     }
     log.info(s"Executing the following counting query on Neo4j: $query")
-    session.run(query, Values.value(Neo4jUtil.paramsFromFilters(filters).asJava))
+    session.readTransaction(
```

Comment:
is there a specific reason to switch to `readTransaction` from `run`?

Reply:
Run(query, value) doesn't accept transaction parameters. I guess I can try replacing `Value` with a map.